source/class/cv/TemplateEngine.js

```diff
@@ -507,7 +531,7 @@ qx.Class.define('cv.TemplateEngine', {
       this.debug("setup");
 
       // login to backend as it might change some settings needed for further processing
-      this.visu.login(true, function () {
+      this.visu.login(true, cv.Config.configSettings.credentials, function () {
```

Comment:
This breaks `client/source/class/cv/io/Client.js` line 367:
```
login : function (loginOnly, callback, context) {
```
where only 3 and not 4 parameters are expected